Susi/Controllers/ChatViewController/ChatVCMethods.swift

```diff
@@ -343,12 +343,17 @@ extension ChatViewController {
     func estimatedFrame(message: String) -> CGRect {
         let size = CGSize(width: 250, height: 1000)
         let options = NSStringDrawingOptions.usesFontLeading.union(.usesLineFragmentOrigin)
-        return NSString(string: message).boundingRect(with: size, options: options, attributes: [NSFontAttributeName: UIFont.systemFont(ofSize: 16)], context: nil)
+        return NSString(string: message).boundingRect(with: size, options: options, attributes: [NSAttributedStringKey.font: UIFont.systemFont(ofSize: 16)], context: nil)
     }
 
     // loads all messages from database
     func loadMessages() {
-        messages = List<Message>(realm.objects(Message.self))
+        let results: Results<Message> = realm.objects(Message.self)
```

Comment:
No need to convert here when the `Results<Message>` object can be used. Instead change the type of messages to this.

Reply:
Okey! Now I declared `messages` as `var messages = [Message]()` and used `messages = Array(realm.objects(Message.self))` in the `loadMessages` function.